pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -73,15 +82,56 @@ func (util *VsphereDiskUtil) CreateVolume(v *vsphereVolumeProvisioner) (vmDiskPa
 	// the values to the cloud provider.
 	for parameter, value := range v.options.Parameters {
 		switch strings.ToLower(parameter) {
-		case "diskformat":
+		case diskformat:
 			volumeOptions.DiskFormat = value
 		case "datastore":
 			volumeOptions.Datastore = value
+		case Policy_HostFailuresToTolerate:
+			if !validateVSANCapability(Policy_HostFailuresToTolerate, value) {
+				return "", 0, fmt.Errorf(`Invalid value for hostFailuresToTolerate in volume plugin %s. 
+				The default value is 1, minimum value is 0 and maximum value is 3.`, v.plugin.GetPluginName())
```

Comment:
U meant the "The default value is 1, minimum value is 0 and maximum value is 3.",

"The value can be either 0 or 1",

"The minimum percentage is 0 and maximum percentage is 100."

and so on.?